curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/64cb3f62b10c336bada1c70c.md

```diff
@@ -24,13 +24,13 @@ assert.match(code, /\bcheckpoints\b\s*\.\s*forEach\s*\(/s);
 You should have a `checkpoint` parameter inside the `forEach` callback function.
 
 ```js
-assert.match(code, /\s*checkpoints\s*\.\s*forEach\s*\(\s*checkpoint\s*=>\s*{/s);
+assert.match(code, /\s*checkpoints\s*\.\s*forEach\s*\(\s*(\(\s*checkpoint\s*\)|checkpoint)\s*=>\s*{/s);
```

Comment:
```suggestion
assert.match(code, /\s*checkpoints\s*\.\s*forEach\s*\(\s*(\(\s*checkpoint\s*\)|checkpoint)\s*=>\s*/s);
```
This one still expects an opening `{` to be present

Reply:
I have a silly question. Can't I make the brace optional?

Reply:
I believe this one satisfies the requirements.

Reply:
Sure, you can make it optional as well. It shouldn't matter where the end of the regex is as long as it's testing the callback parameter.